test/GoodGhosting.test.js

```diff
@@ -1200,6 +1359,44 @@ contract("GoodGhosting", (accounts) => {
             await goodGhosting.redeemFromExternalPool({ from: player1 });
             await truffleAssert.reverts(goodGhosting.adminFeeWithdraw({ from: admin }), "No Fees Earned");
         });
+
+        it("withdraw fees when there's only interest generated by external pool", async () => {
```

Comment:
```suggestion
        it("withdraw fees if there are no winners, but there's interest generated by external pool", async () => {
```

Reply:
@fodisi covered in this commit https://github.com/Good-Ghosting/goodghosting-smart-contracts/pull/114/commits/9cd39affa29ed5af43f8b4a7e53cb749a4a41d87

Reply:
@viraj124 
the commit you referred to covers from a player perspective. my comment is related to the admin perspective when there are no winners. Thanks

Reply:
@fodisi 
also there https://github.com/Good-Ghosting/goodghosting-smart-contracts/blob/refactor/missing-unit-tests/test/GoodGhosting.test.js#L1167